app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -79,7 +79,6 @@ def update_visibility_options(attributes)
 
         # @return [Void]
         def update_workflow_options(attributes)
-          update_permission_template(attributes)
```

Comment:
Do we need to have two methods or can `grant_workflow_roles` and `update_workflow_options` be combined into one method?